app/models/host_status/execution_status.rb

```diff
@@ -49,6 +49,10 @@ def to_label(options = {})
     end
   end
 
+  def status_link
+    Rails.application.routes.url_helpers.job_invocation_path(last_stopped_task.parent_task.job_invocations.first)
```

Comment:
Works well, however it does not check for permissions. User must have `:view_job_invocations` permissions for this job invocation, otherwise we shouldn't make the link clickable. You can either use `authorized` named scope on JobInvocation model or use `User.current.can?` to check that.